cvat/settings/base.py

```diff
@@ -397,10 +397,10 @@ def generate_ssh_keys():
     LOGGING['loggers']['cvat.server']['handlers'] += ['logstash']
     LOGGING['loggers']['cvat.client']['handlers'] += ['logstash']
 
-DATA_UPLOAD_MAX_MEMORY_SIZE = 100 * 1024 * 1024  # 100 MB
-DATA_UPLOAD_MAX_NUMBER_FIELDS = None   # this django check disabled
-LOCAL_LOAD_MAX_FILES_COUNT = 500
-LOCAL_LOAD_MAX_FILES_SIZE = 512 * 1024 * 1024  # 512 MB
+DATA_UPLOAD_MAX_MEMORY_SIZE = int(os.getenv('CVAT_MAX_MEMORY_SIZE', '104857600'))  # 100 MB default
```

Comment:
If we add an env variable for every item in the config file, it will be mess. If you need to change settings, there is a better solution:
- Create your own `mysettings.py` file (name it as you like)
- Inside the `mysettings.py` file import appropriate CVAT configuration (e.g.  `from cvat.settings.production import *`)
- Modify necessary variables (e.g. DATA_UPLOAD_MAX_MEMORY_SIZE)
- Use DJANGO_SETTINGS_MODULE to point to the new settings file for your instance.

The approach is more flexible. You can redefine your settings as you like.

Do you still need the PR or the described approach will work for you?